VideoWeb/VideoWeb.UnitTests/Mappings/ConferenceResponseMapperTests.cs

```diff
@@ -126,6 +126,11 @@ public void Should_map_all_properties_for_more_then_4_participants()
             foreach (var participantResponse in participantsResponse)
             {
                 var position = participantResponse.TiledDisplayName.Split(';');
+                if (participantResponse.Role == Role.Judge)
+                {
+                    participantResponse.TiledDisplayName.StartsWith("T0").Should().BeTrue();
```

Comment:
Could you add this check for less than 4 participants also please?

Reply:
It's done, you could not see just code, there is separate test for less than 4 participants